database/migrations/2023_10_31_204320_create_wikibase_notification_sent_records_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWikibaseNotificationSentRecordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wikibase_notification_sent_records', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+
+            $table->integer('wiki_id')->nullable()->unsigned();
+//
+//            $table->unsignedInteger('wiki_id');
+//            $table->foreign('wiki_id')->references('id')->on('wikis');
```

Comment:
I think we still want this foreign key?

Reply:
I commented them out while debugging, of course we need that